pkg/pointer/pointer.go

```diff
@@ -0,0 +1,30 @@
+// Package pointer provides utilities for pointer handling that aren't avaliable in go.
+// Feel free to add more pointerification functions for more types as you need them.
+package pointer
+
+import "time"
+
+// Duration returns a pointer to its argument
```

Comment:
Please punctuate all the godoc in this file. https://blog.golang.org/godoc-documenting-go-code